static/app/components/tooltip.tsx

```diff
@@ -114,74 +111,14 @@ function isOverflown(el: Element): boolean {
   return el.scrollWidth > el.clientWidth || Array.from(el.children).some(isOverflown);
 }
 
-function getTooltipPortal(useGlobalPortal: boolean): HTMLElement {
-  // Use a single global portal element for tooltips
-  if (useGlobalPortal) {
-    let portal = document.getElementById('tooltip-portal');
-    if (!portal) {
-      portal = document.createElement('div');
-      portal.setAttribute('id', 'tooltip-portal');
-      document.body.appendChild(portal);
-    }
-    return portal;
-  }
-
-  // Each tooltip gets it's own div element
+function createTooltipPortal(): HTMLElement {
   const portal = document.createElement('div');
   document.body.appendChild(portal);
   return portal;
 }
```

Comment:
I think we need to keep around the global portal no? There is no reason to create a new div element in the body for _every_ tooltip, of which there can be hundreds on a single page.

Reply:
Yeah, we can but the DOM is fast and I dont think this will ever be an issue. The right fix would probably be to make this lazy instead of worrying about performance. I'll give it a shot

Reply:
tbf I'm actually not sure what the requirement was for needing each tooltip to have it's own portal element when we're rendering all of them

Reply:
@k-fish

Reply:
I removed the whole tooltip portal logic and just render to the document.body now. Not much different from what we were doing before, just w/o that extra div.

Reply:
@k-fish yea, I actually just dropped the whole tooltip portal thing. Just use document.body for createPortal